shell/platform/darwin/ios/ios_surface.h

```diff
@@ -31,7 +31,10 @@ class IOSSurface {
 
   virtual void UpdateStorageSizeIfNecessary() = 0;
 
-  virtual std::unique_ptr<Surface> CreateGPUSurface() = 0;
+  // Creates a secondary surface using the supplied GrContext, if supported.
+  // The gr_context argument may be nullptr.
```

Comment:
nit: mention that it creates a new GrContext if `gr_context` is null

Reply:
Updated this comment to make more sense.